styles.css

```
@@ -0,0 +1,307 @@
+/* kleuren
+Paars #5722CD:
+Blauw #4845E4:
+Roze #ED4385:
+Zwart #0C0C0C:voor standaard tekst
+Lichtpaars #F2F0FF:
+Lichtgrijs #F3F6F9:
+voor de achtergrond van de projecten:
+Grijs #7A7A7A:
+*/
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+
+ */
+
+/
+/*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+p {
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h2 {
+    font-family: 'Merriweather', serif;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/*position: relative; nog ergens in fietsen, maar waar?*/
+.outer-container {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    border: 2px dashed orange;
+}
+
+.inner-container {
+    display: flex;
+    flex-basis: 800px;
+    flex-shrink: 1;
+    flex-grow: 0;
+    flex-direction: column;
+}
+
+
+.navbar  {
+    border: 2px dashed orange;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    background-color: #5722CD;
+    font-family: 'Merriweather', serif;
+    color: #F3F6F9;
+}
+
+.navbar ul {
+    border: 2px dashed yellow;
+    list-style: none;
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-basis: 400px;
+}
+
+.navbar li {
+    border: 2px dashed deeppink;
+}
+
+header {
+    border: 2px dashed orange;
+    text-align: center;
+    display: flex;
+    background-color: #5722CD;
+    color: #F3F6F9;
+    flex-direction: column;
+    justify-content: center;
+    font-family: 'Merriweather', serif;
+    flex-basis: 400px;
+    padding: 100px 100px;
```

Dit kun je nóg korter schrijven:
```suggestion
    padding: 100px;
```